broadcast-message.php

```diff
@@ -27,14 +27,19 @@
 
 require_once('common.php');
 
-authentication_require_role('global-admin');
+if ($CONF['sendmail_only_global_admin'] == 'YES' ) {
```

Comment:
I've changed that.

Speaking of not migrated files: The indentation of the file looks very scary...